fix(ui): update locale on settings/profile

baymax-codecare opened the pull request:
… off

<!--  Thanks for sending a pull request!
If this is your first time, please read our contributor guidelines: https://github.com/Satellite-im/Core-PWA/wiki/Contributing
-->

**What this PR does** 📖
on settings/profile page, search location, and language multi-select dropdown, when no search result, the message is cut off. put custom no result message instead of default one to fix this issue.
**Which issue(s) this PR fixes** 🔨
AP-310
<!--AP-X-->

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

stavares843 commented:
/rebase

stavares843 reviewed (changes requested):
@baymax-codecare 

thanks for adding

i know you added the locales on the other repo but we removed the submodule so you need to add the locale on the main repo on this file
https://github.com/Satellite-im/Core-PWA/blob/dev/locales/en-US.js

baymax-codecare commented:
> @baymax-codecare
> 
> thanks for adding
> 
> i know you added the locales on the other repo but we removed the submodule so you need to add the locale on the main repo on this file https://github.com/Satellite-im/Core-PWA/blob/dev/locales/en-US.js

updated! :)
